linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -31,6 +31,7 @@ class HttpInitializer extends ProtocolInitializer.Simple {
       .prepend(Headers.Dst.PathFilter.module)
       .replace(StackClient.Role.prepFactory, DelayedRelease.module)
       .prepend(http.ErrorResponder.module)
+      .insertAfter(http.ErrorResponder.role, ResponseFramingFilter.module)
```

Comment:
I think these errors should probably count against failure accrual...  We'd want a load balancer to evict nodes sending these sorts of malformed responses.